string robot_id
bool completed